app/views/admin/link_check_reports/show.js.erb

```diff
@@ -0,0 +1,3 @@
+<% if @report.completed? %>
```

Comment:
If the report hasn't been completed yet should we be showing a progress indicator? Or perhaps this does not deserve to be in this file?